starknet_py/tests/e2e/mock/contracts/universal_deployer.cairo

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.4.0 (utils/presets/UniversalDeployer.cairo)
```

Comment:
We could actually use v0.5.0 (I don't know if anything has changed since v0.4.0, I think not though)